internal/tiltfile2/tiltfile_test.go

```diff
@@ -502,6 +502,18 @@ k8s_resource('bar', 'bar.yaml')
 	}
 }
 
+func TestDockerCompose(t *testing.T) {
```

Comment:
O.o

Reply:
I mean, just the tiltfile part of it! ...point taken tho.